revolute_controllers: [joint1, joint2, joint3, joint4, joint5, joint6, joint7]
prismatic_controllers: [claw]
prismatic_factors: [[-0.0072, -0.005]]

robot_arm_names: [cyton]
cyton_controller_names: [joint1, joint2, joint3, joint4, joint5, joint6, joint7, claw]

